angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -79,15 +79,15 @@ <h4 class='header'>Directory</h4>
       <div ng-click="showingDetails=!showingDetails">
         <p>
           <a href="javascript:;" ng-if="item.titles[0] || item.address">
-            <span ng-if="!showingDetails">See More</span>
-            <span ng-if="showingDetails">See Less</span>
+            <span ng-if="!showingDetails">See more</span>
+            <span ng-if="showingDetails">See less</span>
           </a>
         </p>
       </div>
     </div>
     <div class="seeMoreResults">
-      <p ng-if="wiscDirectoryResultCount>0">
-        <a ng-href="{{$scope.directorySearchUrl + searchTerm}}">See all {{wiscDirectoryResultCount}} directory results</a>
+      <p ng-if="wiscDirectoryResultCount>0" id="wiscDirectorySeeMoreResults">
```

Comment:
Should this be `ng-if="wiscDirectoryResultCount>wiscDirectoryResultLimit"` so that the call to action only displays if the action is meaningful, that is, will show more than is already shown?